src/components/client/common/upcomingEvents/UpcomingEvents.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import './upcomingEvents.css'
+// import Swiper core and required modules
+import { Navigation, Pagination, Scrollbar, A11y } from 'swiper'
+
+import { Swiper, SwiperSlide } from 'swiper/react'
+import heroSecImage from '../../../../assets/images/heroSecImg.svg'
+// Import Swiper styles
+import 'swiper/css'
+import 'swiper/css/navigation'
+import 'swiper/css/pagination'
+import 'swiper/css/scrollbar'
+
+function UpcomingEvents() {
+  return (
+    <div className="container mb-5">
+      <div className="text-center text-color pb-5">
+        <h2>
+          <b>UPCOMING EVENTS</b>
+        </h2>
+      </div>
+      <div className="shadow p-5">
+        <Swiper
+          modules={[Navigation, Pagination]}
+          navigation
+          pagination={{ clickable: true }}
+          onSwiper={(swiper) => console.log(swiper)}
+          onSlideChange={() => console.log('slide change')}>
```

Comment:
why u addded console logs ?

Reply:
i copied this code part directly from swiper documentation and i have no idea why they are here. I removed them now.

Reply:
I think those functions need for the swipe behaviour @sanjanaDinith

Reply:
they are working ayya, i checked